### PR TITLE
Add planout@3.0.1 w/ npm auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #9188
Related issue(s): # #

@cdnjs/intern3 Please help me reviewing this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/HubSpot/PlanOut.js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/planout
 * License and its reference: BSD-3-clause, [ref](https://github.com/HubSpot/PlanOut.js/issues/48)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 38
   - Count of stars: 136
   - Count of forks: 23
 * NPM download stats (optional):
   - Downloads in the last day: 292
   - Downloads in the last week: 1381
   - Downloads in the last month: 4767

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@sufuf3 I add them. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t Please review this PR. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t Sorry that I missed those versions. I added them.

@clairetsai818 The author add new tag on GitHub recently, but still not update on npm. So I think I would still using npm auto-update as it is more versions on npm. 

Please review this PR again. Thanks.

**Comment by pvnr0082t** [comment]
@hare1039 Maybe the author just forgot to publish the latest version on NPM. Please notify the author. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t npm updated! Please review this PR again. Thanks.

**Comment by hare1039** [comment]
ping @clairetsai818

**Comment by clairetsai818** [comment]
LGTM

**Comment by pvnr0082t** [comment]
@hare1039 Please rebase onto the latest master branch, thanks.

**Comment by hare1039** [comment]
@pvnr0082t I've update to latest master branch.

File: ajax/libs/planout/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "keywords": [
+    "experiments",
+    "a/b testing",
+    "multivariate testing",
+    "experiment design",
+    "ab",
+    "split testing"
+  ],
+  "author": "Guy Aridor",
+  "license": "BSD-3-clause",
```
Comment (PeterDaveHello) [inline ajax/libs/planout/package.json]: `BSD-3-Clause`, you should see warning from the `fixFormat` tool.

cc @cdnjs/intern2 @cdnjs/intern3

Comment (hare1039) [inline ajax/libs/planout/package.json]: @pvnr0082t I fix the license field. Please review this PR again. Thanks.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 @cdnjs/intern3 note the title please, this PR contains multiple versions.

**Pull request merged**